azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -20,6 +20,7 @@
 _SERVICE_ENDPOINT_TYPE = [SERVICE_ENDPOINT_TYPE_GITHUB, SERVICE_ENDPOINT_TYPE_AZURE_RM]
 _SERVICE_ENDPOINT_AUTHORIZATION_SCHEME = [SERVICE_ENDPOINT_AUTHORIZATION_PERSONAL_ACCESS_TOKEN,
                                           SERVICE_ENDPOINT_AUTHORIZATION_SERVICE_PRINCIPAL]
+_ACCESS_LEVELS = ['advanced', 'earlyAdopter', 'express', 'none', 'professional', 'stakeholder']
```

Comment:
Is this a fixed list? More specifically is this expected to change in near future?
Is there any way we can query this from some rest api

Reply:
Yes, we can query this from [REST API](https://docs.microsoft.com/en-us/rest/api/azure/devops/memberentitlementmanagement/user%20entitlement%20summary/get?view=azure-devops-rest-5.0) (which returns list of valid licenses/access-level for that account)  . The list seems to be fixed though, that's why adding new command was avoided.
@geverghe

Reply:
If we know this is not going to change in near future than hardcoding is fine, else we should get this from rest api.